src/services/sync/strategies/CSVSyncer.js

```diff
@@ -227,6 +227,7 @@ class CSVSyncer extends mix(Syncer).with(ClassImporter) {
 					if (CSVSyncer.isValidBirthday(mappedRecord.birthday)) {
 						mappedRecord.birthday = CSVSyncer.convertToDate(mappedRecord.birthday);
 					} else {
+						this.logWarning(`Geburtsdatum fehlerhaft: ${record.firstName},${record.lastName},${mappedRecord.birthday}`);
```

Comment:
The combination of this.logWarning and this.stats.errors.push all the time look for me like we want to have a handler that do this 2 thinks with the same data structure. 
this.pushFail(level, type, entity, message) =>
 -  this.logWarning || this.logError
 -  this.state.errors.push 
 - this.stats.users.errors += 1 || this.stats.users.failed+= 1;

Reply:
I just tried. But the error counter works a bit differently in all the Syncers. I can find:
`this.stats.users.failed += 1`
`this.stats.users.errors += 1;`
`this.stats.classes.failed += 1;`
`this.stats.schools.errors += 1;`

I don't really want to change that, because I might break business logic. Unfortunately this only a bit of log improvement, not a rework of the Syncers.